packages/dds/tree/src/simple-tree/api/schemaFactory.ts

```diff
@@ -997,7 +989,7 @@ export class SchemaFactory<
 	 * @returns The structural {@link TreeNodeSchema} associated with the given name and types.
 	 * @throws `UsageError` if a schema structurally named schema with the same name is cached in `structuralTypes` but had different input types.
 	 */
-	private getStructuralType(
+	protected getStructuralType(
```

Comment:
something I noticed: SchemaFactory is @sealed, but SchemaFactoryAlpha is not :0

Reply:
But it's also `@alpha`, so we can make breaking changes at any time 🙂 Still probably a good idea to mark it as `@sealed` to prevent future issues though...